hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -281,6 +281,7 @@ public Response put(
       }
       if (output != null) {
         output.close();
+        getMetrics().updateCreateKeySuccessStats(startNanos);
```

Comment:
This may be executed for both success and failure.  Success stats should be updated inside `if (auditSuccess)`.